proposal/q40603/proposal.md

```diff
@@ -0,0 +1,97 @@
+Proposal for Pairs Trading 
+===========================
+
+A pairs trade is a trading strategy that involves matching a long position with a short position in two stocks with a high correlation. It uses statistical and technical analysis to seek out potential market-neutral profits.
+
+In this term project, I would like to optmize the Pairs Trading computing engine developed by the classmate in my lab. He combined the pairs trading techinque with Taiwan Stock Market intraday high-frequency trading, whcih I think is an opportunity and is suitalbe for NSD term project.
+
+Basic information
+=================
+
+Pairs Trading always comes along with a term "Cointegration". The basic ideas of Cointegration is to test whether two time series move similarly. When applied to stock market, we can find two assets that move with cointegration, assign different weight at the assets respectively, and sell the high priced stock, buy the low priced stock. 
+
+| two stock series | Coinegration series |
+| -------- | -------- |
+| ![](https://i.imgur.com/ajKsdUc.png)     | ![](https://i.imgur.com/2cPbJ7F.png)    |
```

Comment:
If you own the images, it's a good idea to add them here in the proposal directory as records.